studybuddies/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,6 @@
 export const environment = {
-  production: true
+  production: true,
+  name: 'produccion',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
  name: "produccion",
```